autoload/limelight.vim

```diff
@@ -139,7 +139,7 @@ function! s:dim(coeff)
   let fg = synIDattr(synid, 'fg#')
   let bg = synIDattr(synid, 'bg#')
 
-  if has('gui_running') || has('termguicolors') && &termguicolors
+  if has('gui_running') || exists('+termguicolors') || has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
```

Comment:
Any reason to change `has('termguicolors') && &termguicolors` to `exists('+termguicolors')`? Shouldn't we check the value of `&termguicolors`? It can be either on or off.